.github/workflows/trigger-gitlab.yml

```diff
@@ -76,6 +76,7 @@ jobs:
 
       - name: Push to gitlab
         run: |
+          set -x
           mkdir -p ~/.ssh
           echo "${SCHUTZBOT_SSH_KEY}" > ~/.ssh/id_rsa
```

Comment:
Are we sure github will mask the private key here?

Reply:
Huh, good question, let my try in my testing repo.

Reply:
I defined it as repository secret, that's not going to work?

Reply:
change 
```
    env:
      SCHUTZBOT_SSH_KEY: I_DONT_CARE
```
to 
```
    env:
      SCHUTZBOT_SSH_KEY: ${{ secrets.SECRET }}
```

Reply:
cool, seems alright